docs/openapi/docs.go

```diff
@@ -0,0 +1,163 @@
+                "consumes": [
+                    "*/*"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "tags": [
+                    "Schema Review"
+                ],
+                "summary": "Check the SQL statement.",
+                "parameters": [
+                    {
+                        "type": "string",
+                        "description": "The environment name",
+                        "name": "env",
+                        "in": "query",
+                        "required": true
+                    },
+                    {
+                        "type": "string",
+                        "description": "The SQL statement",
+                        "name": "statement",
+                        "in": "query",
+                        "required": true
+                    },
+                    {
+                        "enum": [
+                            "MySQL",
+                            "PostgreSQL",
+                            "TiDB"
+                        ],
+                        "type": "string",
+                        "description": "The database type",
+                        "name": "database",
+                        "in": "query",
+                        "required": true
+                    },
+                    {
+                        "type": "string",
+                        "default": "utf8mb4",
+                        "description": "The database charset. Default 'utf8mb4'",
+                        "name": "charset",
+                        "in": "query"
+                    },
+                    {
+                        "type": "string",
+                        "default": "utf8mb4_general_ci",
+                        "description": "The database collation. Default 'utf8mb4_general_ci'",
+                        "name": "collation",
+                        "in": "query"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/api.SQLCheckResult"
+                        }
+                    },
+                    "400": {
+                        "description": "Bad Request",
+                        "schema": {
+                            "$ref": "#/definitions/echo.HTTPError"
+                        }
+                    },
+                    "500": {
+                        "description": "Internal Server Error",
+                        "schema": {
+                            "$ref": "#/definitions/echo.HTTPError"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "advisor.Advice": {
+            "type": "object",
+            "properties": {
+                "code": {
```

Comment:
nit: list the field order according to the type definition in the code

Reply:
The files under `openapi` folder will always be re-generated by the swagger cmd, we'd better not edit it manually. Will find if exist other ways to consequence them.